From 90b22572e7608447d1161bffd7218a844baca9dd Mon Sep 17 00:00:00 2001
From: Nick Richards <nick@nedrichards.com>
Date: Mon, 26 Aug 2019 13:28:31 +0100
Subject: [PATCH] Add OARS rating

Add OARS rating that allows parental ratings in app stores https://hughsie.github.io/oars/
---
 data/com.uploadedlobster.peek.appdata.xml.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/data/com.uploadedlobster.peek.appdata.xml.in b/data/com.uploadedlobster.peek.appdata.xml.in
index 4476459..4142c58 100644
--- a/data/com.uploadedlobster.peek.appdata.xml.in
+++ b/data/com.uploadedlobster.peek.appdata.xml.in
@@ -80,6 +80,8 @@
 
   <developer_name>Philipp Wolfer</developer_name>
   <update_contact>ph.wolfer@gmail.com</update_contact>
+  
+  <content_rating type="oars-1.1" />
 
   <releases>
     <release version="1.3.1" date="2018-03-29" />
